evalml/pipelines/components/transformers/preprocessing/featuretools.py

```diff
@@ -0,0 +1,67 @@
+from featuretools import EntitySet, calculate_feature_matrix, dfs
+
+from evalml.pipelines.components.transformers.transformer import Transformer
+from evalml.utils.gen_utils import (
+    _convert_to_woodwork_structure,
+    _convert_woodwork_types_wrapper
+)
+
+
+class FeatureTools(Transformer):
+    """Featuretools component that generates features for ww.DataTables and pd.DataFrames"""
+    name = "Featuretools"
+    hyperparameter_ranges = {}
+
+    def __init__(self, max_depth=2, random_state=0, **kwargs):
+        """Allows for featuretools to be used in EvalML
+
+        Arguments:
+            max_depth (int): The max allowed depth of features. Defaults to 2
+            random_state (int, np.random.RandomState): seed for the random number generator
+        """
+        parameters = {"max_depth": max_depth}
+        self._ft_es = EntitySet()
```

Comment:
Why not define this within `_make_entity_set` and set `_make_entity_set` as static?